oa/INSTALL.md

```diff
@@ -0,0 +1,132 @@
+         Create a csv file with ip ranges in integer format and give the coordinates for each range.
+     
+    2.3. Add a file networkcontext.csv: Ip names file used by OA when running data type DNS and Proxy. This file
+         should contains two columns, one for Ip the other for the name, example: 
+    
+            10.192.180.150, AnyMachine
+            10.192.1.1,     MySystem
+            
+    For more information on context files go to [oni-oa/context/README.md](https://github.com/Open-Network-Insight/oni-oa/blob/1.1/README.md)
+            
+ 3. oni-setup project contains scripts to install hive database but also includes the main configuration file for this tool.
+    That file is called duxbay.con contains different variables user can set up to customize their installation, in fact, some
+     of them are required to be updated in order to have oni-ml and oni-oa working.
+     
+     To run the OA process it's required to install oni-setup. If it's already installed just make sure the following configuration are set up in duxbay.conf file.
+    
+        LUSER: represents the home folder for the user in the Machine Learning node. It's used to know where to return feedback.
+        HUSER: represents the HDFS folder. It's used to know from where to get Machine Learning results.
+        IMPALA_DEM: represents the node running Impala demon. It's needed to execute Impala queries in the OA process.
+        DBNAME: Hive database, the name is required for OA to execute queries against this database.
+        LPATH: represents the local path where the feedback is going to be sent, it actually works with LUSER.
+    
+ 4. Configure components. Components are python modules included in this project that add context and detail to the data 
+    being analyzed. There are five components and while not all components are required to every data type, it's recommended to
+    configure all of them in case new data types are analyzed in the future.
+    For more details about how to configure each component go to [oni-oa/oa/components/README.md](https://github.com/Open-Network-Insight/oni-oa/blob/1.1/oa/components/README.md).
+    
+ ####Usage
+ 
+ Operational Analytics process is triggered with the execution of start_oa.py. This Python script will execute the OA process
+  for only one data type (Flow, DNS or Proxy) at a time. If users need to process multiple data types at the same time, multiple
+  instances of the same script need to be executed. 
+  
+  To execute Operational Analytics in ONI 1.1 go to folder oni-oa/oa and execute the following command:
+        
+        [solution-user@edge-server] python2.7 start_oa.py -d YYYYMMDD -t [flow, dns, proxy] -l N
+    
+  Example
+  
+        [oniuser@node03] python2.7 start_oa.py -d 20160827 -t flow -l 3000
+        
+  Parameters
+ 
+        -d Date for the machine learning results that ara going to be processed. Format shouldbe YYYY (year in 4 digits)
```

Comment:
Date for the machine learning results that are going to be processed. Format should be YYYY (year in 4 digits)